### DISCUSSION TITLE
Customizing Azuracast and Live broadcast start messages

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#6384] [Q&A]
I've read in the forum (can't recall exactly where) our main man Buster saying that such messages could be customized on a 'per station' basis.  I've since scoured the dashboard with a magnifying glass, looking for a menu to do so, all without any luck. Seems to me the "Custom Branding" module would have been the perfect place for such a function...
The following piece of coding is found in LS Config. and appears to be what needs to be overridden to effect such custom messages.
-----------------------------------------------------
 [# Live Broadcasting
live = input.harbor("/", id = "input_streamer", port = 8005, auth = dj_auth, icy = true, icy_metadata_charset = "UTF-8", metadata_charset = "UTF-8", on_connect = live_connected, on_disconnect = live_disconnected, buffer = 30.00, max = 35.00)

def insert_missing(m) =
    if m == [] then
        [("title", "Live Broadcast"), ("is_live", "true")]
    else
        [("is_live", "true")]
    end]- Live Broadcasting
live = input.harbor("/", id = "input_streamer", port = 8005, auth = dj_auth, icy = true, icy_metadata_charset = "UTF-8", metadata_charset = "UTF-8", on_connect = live_connected, on_disconnect = live_disconnected, buffer = 30.00, max = 35.00)

def insert_missing(m) =
    if m == [] then
        [("title", "Live Broadcast"), ("is_live", "true")]
    else
        [("is_live", "true")]
    end]
-------------------------------------------------------
Do we need to write a custom script to override this or is there a command module I haven't found yet? 
Our station caters to a franco audience and we would love to have such things as "[djNAME] est en ondes (_is live_)" at the start of a live broadcast and "Bienvenue sur (_Welcome to_) [station name]. [station motto]" at restart of broadcast.

**Comment by renatopinto89** [comment]
I´m also interested in this, since some of our DJs don´t send metadata, and when they try to send any custom data, depending on the software they´re using, it will "get stuck" on "Live Broadcast"

It would be nice to have some way to customize that.

Thank you in advance
Kind Regards

**Comment by BusterNeece** [comment]
In the latest Rolling Release, you can now specify a custom text for the "Live Broadcast" message (as well as custom "Station Offline" text); find the new field in the Streamers/DJs tab of the station profile.

**Reply by SpeedyCharly** [reply]
Found Station Offline in Branding Settings module... but haven't found Live Broadcast anywhere...

**Reply by BusterNeece** [reply]
Should be here on the Station Profile edit page.

<img width="925" alt="Screenshot 2023-08-12 113932" src="https://github.com/AzuraCast/AzuraCast/assets/6744885/809fd994-0cbc-4daa-84aa-1cd2af035567">
